adops/gam-creative-banner-sbs.md

```diff
@@ -0,0 +1,110 @@
+These instructions assume you're using the Prebid Universal Creative (PUC). See the [Prebid Universal Creative](/overview/prebid-universal-creative.html) documentation for alternate approaches.
+
+{:start="6"}
+6. Select **Standard** as the **Code type**.
+
+{: .alert.alert-info :}
+**AMP**: If you are using AMP, you should still select **Standard** as the code type. The “AMP” option is for AMPHTML hosted by a 3rd party.
+
+{:start="7"}
+7. Enter one of the scripts shown below, depending on whether Prebid is configured to send all bids or only the top price bid.
+
+**Send All Bids Configuration**
+
+{: .alert.alert-warning :}
+Be sure to replace BIDDERCODE with the appropriate bidder. For example, if the bidder code is `BidderA`, the `adid` would be `%%PATTERN:hb_adid_BidderA%%`.
+
+```
+    <script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js"></script>
+    <script>
+      var ucTagData = {};
+      ucTagData.adServerDomain = "";
+      ucTagData.pubUrl = "%%PATTERN:url%%";
+      ucTagData.adId = "%%PATTERN:hb_adid_BIDDERCODE%%";
+      ucTagData.cacheHost = "%%PATTERN:hb_cache_host_BIDDERCODE%%";
+      ucTagData.cachePath = "%%PATTERN:hb_cache_path_BIDDERCODE%%";
+      ucTagData.uuid = "%%PATTERN:hb_cache_id_BIDDERCODE%%";
+      ucTagData.mediaType = "%%PATTERN:hb_format_BIDDERCODE%%";
+      ucTagData.env = "%%PATTERN:hb_env%%";
+      ucTagData.size = "%%PATTERN:hb_size_BIDDERCODE%%";
+      ucTagData.hbPb = "%%PATTERN:hb_pb_BIDDERCODE%%";
+      // mobileResize needed for mobile GAM only
+      ucTagData.mobileResize = "hb_size:%%PATTERN:hb_size_BIDDERCODE%%";
+      try {
+        ucTag.renderAd(document, ucTagData);
+      } catch (e) {
+        console.log(e);
+      }
+    </script>
+```
+
+{: .alert.alert-danger :}
+Warning: Be sure none of the attribute names are longer than 20 characters. See [Send All Bids Key Value Pairs](/adops/send-all-vs-top-price.html#key-value-pairs) for more information.
+
+**Send Top Price Bid Configuration**
+
+```
+<script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/creative.js"></script>
+<script>
+  var ucTagData = {};
+  ucTagData.adServerDomain = "";
+  ucTagData.pubUrl = "%%PATTERN:url%%";
+  ucTagData.targetingMap = %%PATTERN:TARGETINGMAP%%;
+  ucTagData.hbPb = "%%PATTERN:hb_pb%%";
+
+  try {
+    ucTag.renderAd(document, ucTagData);
+  } catch (e) {
+    console.log(e);
+  }
+</script>
+```
+
+{:start="8"}
+8. Select whether to **Serve into a SafeFrame**. See [Creative Considerations](/adops/creative-considerations.html) for more on using SafeFrames.
+
+{: .alert.alert-info :}
+**AMP**: For AMP, you must select **Serve into a SafeFrame**.
+
+Your creative settings will look something like this. In this example we're assuming a Send All Bids configuration, and we’ve replaced BIDDERCODE with the code for our bidder, BidderA.
+
+![Banner creative settings](/assets/images/ad-ops/gam-sbs/banner-creative-settings.png)
```

Comment:
Some of these values end up being longer than 20 characters. Would it be good to show how some of them get truncated in the screenshot, or utilize a bidder name that would not be truncated? A note could be added to further reinforce how long values are truncated.